packages/backend/src/launch-vm.ts

```diff
@@ -0,0 +1,185 @@
+import path from 'node:path';
+import * as extensionApi from '@podman-desktop/api';
+import { isMac } from './machine-utils';
+import fs from 'node:fs';
+
+// Ignore the following line as this is where we will be storing the pid file
+// similar to other projects that use pid files in /tmp
+// eslint-disable-next-line sonarjs/publicly-writable-directories
+const pidFile = '/tmp/qemu-podman-desktop.pid';
+
+// Must use "homebrew" qemu binaries on macOS
+// as they are found to be the most stable and reliable for the project
+// as well as containing the necessary "edk2-aarch64-code.fd" file
+// it is not advised to use the qemu binaries from qemu.org due to edk2-aarch64-code.fd not being included.
+const macQemuArm64Binary = '/opt/homebrew/bin/qemu-system-aarch64';
+const macQemuArm64Edk2 = '/opt/homebrew/share/qemu/edk2-aarch64-code.fd';
+const macQemuX86Binary = '/opt/homebrew/bin/qemu-system-x86_64';
```

Comment:
does it mean, it'll work only on mac/Silicon and not mac/Intel ?

Reply:
also we recommend to not install podman using brew but here it'll install qemu using brew ?!

Reply:
@benoitf Yes, it will only work with mac M1 as that's what we're supporting at the moment.

The reasoning for brew install is that is the recommended install pattern by podman-bootc (https://github.com/containers/podman-bootc), vfkit as well as the qemu project: https://www.qemu.org/download/#macos

Downloading the binaries directly would not be good as there are *a lot* that's needed / added (qemu-img, qemu-io, qemu-user-static, etc.):

```sh
▶ qemu-
qemu-edid                 qemu-system-loongarch64   qemu-system-riscv32
qemu-img                  qemu-system-m68k          qemu-system-riscv64
qemu-io                   qemu-system-microblaze    qemu-system-rx
qemu-nbd                  qemu-system-microblazeel  qemu-system-s390x
qemu-storage-daemon       qemu-system-mips          qemu-system-sh4
qemu-system-aarch64       qemu-system-mips64        qemu-system-sh4eb
qemu-system-alpha         qemu-system-mips64el      qemu-system-sparc
qemu-system-arm           qemu-system-mipsel        qemu-system-sparc64
qemu-system-avr           qemu-system-nios2         qemu-system-tricore
qemu-system-cris          qemu-system-or1k          qemu-system-x86_64
qemu-system-hppa          qemu-system-ppc           qemu-system-xtensa
qemu-system-i386          qemu-system-ppc64         qemu-system-xtensaeb
```